src/references/cli/epinio_app_port-forward.md

```diff
@@ -0,0 +1,32 @@
+## epinio app port-forward
+
+creates a shell to the application
```

Comment:
Shell to the application ? I thought this is about forwarding ports in the system ?

Edit: This looks to be copy/pasta issue in the sources for the command itself.

Reply:
AH, good catch. Fixed pushing the fix in this PR: https://github.com/epinio/epinio/pull/1198